tests/std/tests/VSO_0000000_type_traits/test.cpp

```diff
@@ -851,6 +858,14 @@ enum LLEnum : long long { xLongExample, yLongExample };
 enum class ExampleEnumClass { xExample, yExample };
 enum class LLEnumClass : long long { xLongExample, yLongExample };
 
+#if _HAS_CXX23
+STATIC_ASSERT(!is_scoped_enum<ExampleEnum>::value);
+STATIC_ASSERT(!is_scoped_enum_v<ExampleEnum>);
+STATIC_ASSERT(!is_scoped_enum_v<test_abc1>);
+STATIC_ASSERT(is_scoped_enum_v<ExampleEnumClass>);
+STATIC_ASSERT(is_scoped_enum_v<LLEnumClass>);
+#endif // _HAS_CXX23

```

Comment:
Could we please add https://github.com/microsoft/STL/pull/1950/files#r645227756 as a test case?